awesome.c

```diff
@@ -84,10 +84,16 @@ init_rng(void)
     lua_getfield(L, -1, "randomseed");
 
     /* Push a seed */
-    lua_pushnumber(L, g_random_int() + g_random_double());
+    lua_pushnumber(L, g_random_int());
```

Comment:
This is the fix that was suggested by @Elv13. Other suggestions (https://github.com/awesomeWM/awesome/pull/3124#issuecomment-667464477) had different approach. I don't know what's best.